puppersim/reacher/reacher_env.py

```diff
@@ -1,112 +1,149 @@
 import pybullet
 import puppersim.data as pd
+from pybullet_utils import bullet_client
+from reacher_kinematics import calculate_forward_kinematics_robot
```

Comment:
`from puppersim import reacher_kinematics`
then do `reacher_kinematics.calculate_forward_kinematics_robot`

Reply:
In general when you import code, you don't want to import the functions directly. You instead want to refer to functions like thefilewhereitsfrom.thefunction

Reply:
ok just fixed this!